subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/FileSystemWatchingBuildActionRunner.java

```diff
@@ -55,29 +56,28 @@ public Result run(BuildAction action, BuildController buildController) {
         DirectorySnapshotterStatistics.Collector directorySnapshotterStatisticsCollector = services.get(DirectorySnapshotterStatistics.Collector.class);
         BuildOperationRunner buildOperationRunner = services.get(BuildOperationRunner.class);
 
-        boolean watchFileSystem = startParameter.isWatchFileSystem();
+        WatchMode watchFileSystemMode = startParameter.getWatchFileSystemMode();
         VfsLogging verboseVfsLogging = startParameter.isVfsVerboseLogging()
             ? VfsLogging.VERBOSE
             : VfsLogging.NORMAL;
         WatchLogging debugWatchLogging = startParameter.isWatchFileSystemDebugLogging()
             ? WatchLogging.DEBUG
             : WatchLogging.NORMAL;
 
-        logMessageForDeprecatedWatchFileSystemProperty(startParameter);
         logMessageForDeprecatedVfsRetentionProperty(startParameter);
-        LOGGER.info("Watching the file system is {}", watchFileSystem ? "enabled" : "disabled");
-        if (watchFileSystem) {
+        LOGGER.info("Watching the file system is {}", watchFileSystemMode.getDescription());
```

Comment:
:o: Do we need to keep it at info?

Reply:
Are you saying we should move this to `DEBUG`?

Reply:
Yes, there's no reason, now that it's out of incubation, to log that a feature is enabled. It's good when you have incubating features, but if it works as intended this is just noise.

Reply:
We actually have some tests that rely on it. I'd leave this in for now.

Reply:
I think it is fine to leave this in info log. I think for info log this one message is interesting and not too verbose. If we want to move it to debug, the tests can use debug logging as well.